### PR TITLE
feat: add release doc

### PR DESCRIPTION
**Pull request by sailorlqh**
This is part of the fix for [issue#2](https://github.com/abcxyz/jvs-plugin-github/issues/21)

### CODE REVIEW

File: doc/release.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+- If a new JVS base image is needed, update [Dockerfile](https://github.com/abcxyz/jvs-plugin-github/blob/main/Dockerfile)
+to point to a new JVS base image, and merge the PR.
+
+- Send a PR to update all dependencies For Go
+```sh
+go get -u && go mod tidy
+```
+
+- Create a tag using `.github/workflows/create-tag.yml` on default branch and
```
Comment (drevell) [inline doc/release.md]: Perhaps link to https://github.com/abcxyz/jvs/actions/workflows/create-tag.yml here

File: doc/release.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+     run the workflow with below inputs.
+
+  - tag name with format `v0.x.x`, using semantic versioning.
```
Comment (drevell) [inline doc/release.md:21]: Maybe `v0` should be changed. We won't be on major version 0 forever.

**Review by drevell — DISMISSED** [review]

File: doc/release.md (hunk 1 above)
Comment (sailorlqh) [reply doc/release.md]: Added, thanks for catching this.

File: doc/release.md (hunk 2 above)
Comment (sailorlqh) [reply doc/release.md:21]: changed to v1.x.x

File: doc/release.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+- Send a PR to update all dependencies For Go
+```sh
+go get -u && go mod tidy
+```
+
+- Create a tag using [create-tag workflow](https://github.com/abcxyz/jvs-plugin-github/blob/main/.github/workflows/create-tag.yml) on default branch and
+     run the workflow with below inputs.
+
+  - tag name with format `v1.x.x`, using semantic versioning.
```
Comment (sqin2019) [inline doc/release.md]: nit: v0.x.x ?

**Review by sqin2019 — DISMISSED** [review]

File: doc/release.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+- If a new JVS base image is needed, update [Dockerfile](https://github.com/abcxyz/jvs-plugin-github/blob/main/Dockerfile)
+to point to a new JVS base image, and merge the PR.
+
+- Send a PR to update all dependencies For Go
+```sh
+go get -u && go mod tidy
+```
+
+- Create a tag using [create-tag workflow](https://github.com/abcxyz/jvs-plugin-github/blob/main/.github/workflows/create-tag.yml) on default branch and
```
Comment (sqin2019) [inline doc/release.md]: https://github.com/abcxyz/jvs-plugin-github/actions/workflows/create-tag.yml is better

Comment (sailorlqh) [reply doc/release.md]: Done

File: doc/release.md (hunk 3 above)
Comment (sailorlqh) [reply doc/release.md]: Done

File: doc/release.md (hunk 2 above)
Comment (sqin2019) [reply doc/release.md:21]: Missed Dave's comment, I am ok with `vx.x.x`

**Review by sqin2019 — APPROVED** [review]

**Review by drevell — APPROVED** [review]

**Pull request merged**